src/Api/Controller/ShowDiscussionController.php

```diff
@@ -98,9 +99,9 @@ protected function data(ServerRequestInterface $request, Document $document)
             $this->includePosts($discussion, $request, $postRelationships);
         }
 
-        $discussion->load(array_filter($include, function ($relationship) {
+        $this->loadRelations(new Collection([$discussion]), array_filter($include, function ($relationship) {
```

Comment:
I'm not sure that I like creating the collection here, it might be better to modify `loadRelations` to accept either a single model or a collection of models, as they both have the same methods, though I dislike that as well, I wish there was a common interface for this, there is a trait, but I don't think you can use traits for typing.

Reply:
Let's keep it like this for now; it's not perfect, but it's good enough.

 re: typing, we can figure out the PHPStan implementation later.